pkg/agent/api_docs.go

```diff
@@ -107,6 +107,14 @@ func getProcessableMappingsFromSnapshot(snapshot *snapshotTypes.Snapshot) []*v3a
 	return processableMappings
 }
 
+// scrape will take care of fetching OpenAPI documentation from each of the
+// AmbassadorMappings resources as we process a snapshot.
+//
+// Be careful as there is a very similar implementation of this logic in the DevPortal which
+// uses the ambassador diag representation to retrieve OpenAPI documentation from
+// Mapping resources.
+// Since both the DevPortal and the agent make use of this `docs` property, evolutions
```

Comment:
should we make that warning on the `docs` property on the CRD as well?

Reply:
Done!